docs/src/content/authors/timothy-alcaide.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Timothy Alcaide",
+  "github": "https://github.com/alcaidio",
+  "youtube": "https://www.youtube.com/@timothyalcaide",
```

Comment:
There is no youtube yet. but you can create it in a follow up PR.

Reply:
Added some code to add youtube but not sure